doc/source/whatsnew/v0.20.0.rst

```diff
@@ -919,11 +925,23 @@ Previous behavior:
 
 New behavior:
 
-.. ipython:: python
+.. code-block:: python
 
-   df.sort_index()
-   df.sort_index().index.is_lexsorted()
-   df.sort_index().index.is_monotonic
+   In [94]: df.sort_index()
+   Out[94]:
+         value
+   a aa      2
+   bb      1
+   b aa      4
+   bb      3
```

Comment:
something went wrong with pasting the indents here, will fix it in the next commit